src/components/pages/Photos/elements/cameraNavbar/CameraNavbar.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import RoverCameraButton from './RoverCameraButton';
+import './CameraNavbar.css';
+
+export default function Navbar() {
```

Comment:
This should be called `CameraNavbar` - we already have a generic `Navbar` component at the top of the page!